web/client/components/map/BaseMap.jsx

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
Can this component replace this one: https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/Map.jsx#L119 ?

Reply:
Yes I developed it also for this purpose. It's a simplified and more clear version of it. Some property mapping, transformation should be enough to replace it in the future. 
Didn't want to replace a so important element at this stage.

Reply:
I would go for replace it now, unless we want to replace it "mai"

Reply:
Yes, I know. But now base map has a minimal support and is not enough tested. if I replace it I've got to fully support cesium and other minor things that are out of scope.

Reply:
we could open a separate issue for that